roles/ssh_jumper/tasks/manage_ssh_jumper_entry.yml

```diff
@@ -0,0 +1,55 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Make sure ~/.ssh/cifw_ssh_config.d directory exists
+  delegate_to: "{{ _config.target | default(cifmw_ssh_jumper_defaults.target) }}"
```

Comment:
so I got confused by the presence here of the delegate_to.

Since we're pushing configurations on 3 locations, it means well have to generate the dataset 3 times, correct?

Reply:
Yes, but the dataset is not the same for all 3 locations. I think the dataset is the same for controller-0 and "hypervisor" - but on localhost we need the proxy_host.

Reply:
right. well, we might go as-is and see to refine it if needed.